### PR TITLE
OPSEXP-2921 Add ACS version 7.3

### PR DESCRIPTION
**Pull request by pmacius**
### Description

Added ACS 7.3 build and test support

### Related Issue

OPSEXP-2921

### Checklist

- [ ] My code follows the project's coding standards.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: repository/Dockerfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ RUN sed -i 's|../modules/platform|modules/platform|' ${CATALINA_HOME}/conf/Catal
 RUN sed -i \
   -re "s|(appender.rolling.fileName\=)(alfresco.log)|\1${CATALINA_HOME}/logs\/\2|" \
   -re "s|(appender.rolling.filePattern=)(alfresco.log.%d\{yyyy-MM-dd\})|\1${CATALINA_HOME}/logs\/\2|" \
-  ${CATALINA_HOME}/webapps/alfresco/WEB-INF/classes/log4j2.properties
+  ${CATALINA_HOME}/webapps/alfresco/WEB-INF/classes/log4j*.properties
```
Comment (pmacius) [inline repository/Dockerfile:32]: Any suggestions how to get rid of repo community on version 7.3 so this change wont be necessary?

File: .github/workflows/cleanup_images.yml [hunk 2]
```diff
@@ -61,6 +61,7 @@ jobs:
           packages: ${{ env.PACKAGE_NAMES }}
           delete-tags: ${{ env.PR_TAGS }}
           dry-run: false
+          use-regex: true
```
Comment (gionn) [inline .github/workflows/cleanup_images.yml:64]: is the plain wildcard `*` not working?

File: repository/Dockerfile (hunk 1 above)
Comment (gionn) [reply repository/Dockerfile:32]: what's inside the 7.3 zip instead of log4j2?

File: test/73-override.yaml
Comment (gionn) [inline test/73-override.yaml:1]: isn't this file the same of 74?

File: repository/Dockerfile (hunk 1 above)
Comment (pmacius) [reply repository/Dockerfile:32]: log4j.properties

File: .github/workflows/cleanup_images.yml (hunk 2 above)
Comment (pmacius) [reply .github/workflows/cleanup_images.yml]: Its working sorry it is some kind of leftover.

File: test/73-override.yaml
Comment (pmacius) [reply test/73-override.yaml:1]: Yeah those are the same

Comment (gionn) [reply test/73-override.yaml:1]: I remember you are also already explicitly linking the acs version to build to the override file to use (without using a variable in the file name for example).

So you could rename this to `override-solr-search.yaml` (which I guess it's the reason to exist) and use it for 7.4 and 7.3

**Pull request merged**